CPT331.iOS/CPT331.iOS/GeocodedPlacemark.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  GeocodedPlacemark.swift
+//  CPT331.iOS
+//
+//  Created by Peter Weller on 19/09/2016.
+//  Copyright © 2016 Peter Weller. All rights reserved.
+//
+
+import Foundation
+import MapboxGeocoder
+
+extension GeocodedPlacemark {
+    func distanceFrom(target: CLLocation) -> Double{
```

Comment:
Is it worth refactoring this function and the one above into one place for reuse and maintenance?